scripts/polkadot-launch/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "composable": "rm -rf /tmp/polkadot-launch && polkadot-launch composable.json",
+    "composable_docker": "(touch 9988.log && tail -f 9988.log > /dev/fd/2 &) && polkadot-launch composable.json",
```

Comment:
not sure, are not docker already can do file from console? also, can we do both via tee? and not sure, are all logs redirected? would prefer all, stderr and stdout.